api/src/services/cereNetworkService.js

```diff
@@ -1,15 +1,26 @@
 const { ApiPromise, WsProvider, Keyring } = require("@polkadot/api");
-const { config, CERE_MAINNET, CERE_MAINNET_WS_PROVIDER_URL_DEFAULT, MNEMONIC_EMPTY } = require("../constants/config");
-const {
-  NETWORKS,
-  CERE_WS_PROVIDER_URL
-} = process.env;
+const { cereTypes, blockchains } = require("../config");
+const { NETWORKS } = process.env;
 
 const networkParams = new Map();
 
 async function init() {
-  networkParams.set(CERE_MAINNET, await initNetwork(CERE_WS_PROVIDER_URL || CERE_MAINNET_WS_PROVIDER_URL_DEFAULT, MNEMONIC_EMPTY));
+  const promises = [];
+  const cere = blockchains[0];
```

Comment:
We can not be sure. Please, update the logic.

Reply:
resolved

Reply:
How? I still see that you suppose that cere blockchain has index `0`

Reply:
resolved